packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -96,7 +96,9 @@ const AppContents = ({
             <DesktopWrapper>
                 <RedirectNoticeModal is_logged_in={is_logged_in} is_eu={is_eu} />
                 {/* Calculate height of user screen and offset height of header and footer */}
-                <ThemedScrollbars height='calc(100vh - 84px)'>{children}</ThemedScrollbars>
+                <ThemedScrollbars height='calc(100vh - 84px)' has_horizontal>
```

Comment:
Why allow horizontal scrolling for `app-contents`?
This will affect all platforms.

Reply:
I think it's better to allow horizontal scrolling for app-contents since it will fix future potential issues on all pages, rather than adding a separate horizontal scroll for each page. Do we need to perform testing on a wider scope then?

Reply:
Yes, since the `app-contents` is the container for all of our platforms / routes loaded by `core` we should have the testing performed on a wider scope.